Project.toml

```diff
@@ -1,5 +1,5 @@
 name = "SparseArrays"
-uuid = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
+uuid = "3f01184e-e22b-5df5-ae63-d93ebab69eaf"
```

Comment:
Eeeh? This need to be changed back.

Reply:
That wasn't working.

Reply:
I don't understand how this works. Whatever was originally there didn't work and the new thing did. Would be nice if you can fix it to what it needs to be.

Reply:
It is wrong here: https://github.com/JuliaLang/SparseArrays.jl/blob/af053194a98848c31ae602cd5ce7f40265b84a8c/.ci/test_and_change_uuid.jl#L10